src/__experimental__/components/form-field/form-field.style.js

```diff
@@ -3,8 +3,16 @@ import BaseTheme from '../../../style/themes/base';
 import { THEMES } from '../../../style/themes';
 
 const FormFieldStyle = styled.div`
-  display: flex;
-  flex-wrap: wrap;
+  ${({ inline }) => {
```

Comment:
What do you think about:
```js
 ${({ inline }) => inline && css`
   display: flex;
   align-items: baseline;
`}
```
Most of the time I saw that above way in the code

Reply:
when written like that it overrides the display property, when you use return you just render what you need

Reply:
Do we need to have more than one display property here?

Reply:
yes, we do to match component's previous behaviour